### PR TITLE
add dob,location indices for person_cases v3 UCR

### PR DESCRIPTION
**Pull request by sravfeyn**
FYI, this is one of the easy_wins to address ASR slowness 

@emord 
@snopoke 
buddy @Rohit25negi

### CODE REVIEW

File: custom/icds_reports/ucr/data_sources/person_cases_v3.json [hunk 1]
```diff
@@ -516,6 +513,30 @@
...
+      },
+      {
+        "column_ids": [
+          "block_id",
+          "dob"
+        ]
+      },
+      {
+        "column_ids": [
+          "district_id",
```
Comment (sravfeyn) [inline custom/icds_reports/ucr/data_sources/person_cases_v3.json]: Also adding `district_id` column

**Comment by snopoke** [comment]
how do we roll this out?

**Comment by sravfeyn** [comment]
Ha, I kind of assumed, UCR framework handles deleting and adding indices, I will look into rollout

**Comment by emord** [comment]
This isn't currently documented in HQ, but was in confluence https://confluence.dimagi.com/display/commcarehq/Using+pghero#Usingpghero-Rollingoutsuggestedindexes

The general approach will be:

1. Create each of these indexes concurrently
2. Deploy this branch
3. Remove the old indexes.

**Comment by sravfeyn** [comment]
@emord There are other UCR querys that filter just on location and not on DOB, does this change adversely affect such querys?

**Comment by emord** [comment]
@sravfeyn No. It should not affect that because the location id is the leading column.

**Comment by sravfeyn** [comment]
Adding a check list to keep track of progress
- [ ] Finish index creation on india
- [ ] Finish index creation on ICDS
- [ ] Merge this
- [ ] Delete old indices

**Comment by sravfeyn** [comment]
Created indices using. @emord I have used _custom_index_name function to generate the index name. Can you confirm this can be merged now and we don't need to worry about UCR tables getting rebuilt?

```
CREATE INDEX CONCURRENTLY  "ix_ucr_icds-cas_static-person_cases_v3_2ae0879a_di_de51b" ON "ucr_icds-cas_static-person_cases_v3_2ae0879a" (district_id, dob);
CREATE INDEX CONCURRENTLY  "ix_ucr_icds-cas_static-person_cases_v3_2ae0879a_bl_77249" ON "ucr_icds-cas_static-person_cases_v3_2ae0879a" (block_id, dob);
CREATE INDEX CONCURRENTLY  "ix_ucr_icds-cas_static-person_cases_v3_2ae0879a_su_6e2b8" ON "ucr_icds-cas_static-person_cases_v3_2ae0879a" (supervisor_id, dob);
CREATE INDEX CONCURRENTLY  "ix_ucr_icds-cas_static-person_cases_v3_2ae0879a_aw_9e11b" ON "ucr_icds-cas_static-person_cases_v3_2ae0879a" (awc_id, dob)
```

**Pull request merged**